apps/omg_status/lib/omg_status/sentry_filter.ex

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Status.SentryFilter do
```

Comment:
> Hopefully filtering these won't hide issues in the cluster for other environments.

Is it possible to add environment to this filtering? (to exclude only failing envs)

Reply:
yes, we could. on the other hand, we have alarms that would detect this. so I don't think it's needed....

Reply:
we actually have sets of alarms that would detect lack of RPC activity, datadog counting RPC calls raises monitors, and internal elixir node connection alarm, it seems enough!